acceptance/bundle/deployment/bind/schema/test.toml

```diff
@@ -0,0 +1,3 @@
+Local = false
```

Comment:
What's the reason to disable it locally?

Reply:
I think it's because it requires additional mocks that we don't have. @anton-107 do you have an idea how many endpoints need to be mocked and whether test.toml `[[Server]]` entries would be enough or you need some code in server_test.go?

Reply:
I propose to get to the ability to run this locally in a follow-up PR